homeassistant/components/home_connect/select.py

```diff
@@ -298,6 +299,16 @@ class HomeConnectSelectEntityDescription(SelectEntityDescription):
             for translation_key, value in VARIO_PERFECT_OPTIONS.items()
         },
     ),
+    HomeConnectSelectEntityDescription(
+        key=OptionKey.HEATING_VENTILATION_AIR_CONDITIONING_AIR_CONDITIONER_FAN_SPEED_MODE,
+        translation_key="fan_speed_mode",
+        options=list(FAN_SPEED_MODE_OPTIONS),
+        translation_key_values=FAN_SPEED_MODE_OPTIONS,
+        values_translation_key={
+            value: translation_key
+            for translation_key, value in FAN_SPEED_MODE_OPTIONS.items()
+        },
+    ),
```

Comment:
Does an air conditioning have a climate entity? Why isn't this part of the fan modes?

Reply:
Given that 2025.9 has arrived already, now it would be great to discuss how it should be implemented.
It is any possibility to implement the fan speed on the climate entity the same way than fan modes are added already.
Or maybe I should use custom fan modes from 1 to 100?